server/src/assembly/resources/conf/iotdb-engine.properties

```diff
@@ -319,6 +319,10 @@ timestamp_precision=ms
 # Datatype: int
 # virtual_storage_group_num = 1
 
+# the interval to log recover progress of each vsg when starting iotdb
+# Datatype: int
+# recovery_log_interval_in_ms=60000
```

Comment:
The Default interval is a little too long, I think 5s will be better.

Reply:
He convinced me that 5s is ok.